### ISSUE TITLE
Migrate PHP tests to PHP toolkit repo

### ISSUE THREAD

**Issue by kadler** [IBM/xmlservice#21]
While we're converting the existing PHP tests to not use the PHP toolkit in #20, they may be useful to  https://github.com/zendtech/IbmiToolkit. We should submit them over there.

**Comment by chukShirley** [comment]
We would appreciate them and a build server to run them on for CI builds, thanks! ;)

**Comment by kadler** [comment]
@chukShirley I'm building all the Python tests as unit tests with mocking. Eventually, I'd like some integration tests too, but with mocked unit tests, you can use something like TravisCI which is awesome.

**Comment by chukShirley** [comment]
Agreed, yes. I was specifically requesting an IBM i partition for running integration tests. It'd be useful for all client toolkits, really. I know this probably isn't the place to make the request, though.

**Comment by kadler** [comment]
Yep, I agree that's really needed. I'd love something like TravisCI for open source IBM i projects. Maybe Liam can get his barryCI going.

**Comment by alanseiden** [comment]
Let's pick one test to start with: https://github.com/IBM/xmlservice/blob/master/test/php/test_50100_ibm_db2_io_jvm_ZZJAVA.phpt (RPG Java test)
- Migrate this PHP Toolkit-dependent test to the https://github.com/zendtech/IbmiToolkit repository
- If there is no equivalent XMLSERVICE test to this Java one that does not rely on the PHP Toolkit, create one for this XMLSERVICE repository.

**Comment by alanseiden** [comment]
Note: There is a second PHP Toolkit - dependent test calling ZZJAVA.
https://github.com/IBM/xmlservice/blob/master/test/php/test_50110_ibm_db2_io_jvm_ZZJAVA2.phpt

**Comment by alanseiden** [comment]
Here is an example of an XMLSERVICE test that does not need the PHP Toolkit. While written in PHP, it does NOT require the toolkit as a dependency. The test passes XML directly to the XMLSERVICE stored procedures.
https://github.com/IBM/xmlservice/blob/master/test/php/test_10120_ZZCALL_ibm_db2_io_pgm.phpt
